index.html

```diff
@@ -569,7 +572,7 @@ <h6 class="my-3">Quality Product</h6>
               </p>
             </div>
           </div>
-          <div class="col-3">
+          <div class="col-md-3">
             <div class="layer-2 text-center">
               <img src="img/24-hours-support 1.png" alt="" />
               <h6 class="my-3">24/7 Support</h6>
```

Comment:
The blog card have a lot of issues

- Pink one is an hover state.
- There should be anchors in the following: 
     1- Image
     2- Title
     3- Read More
- Content padding should be handled from one place, you shouldn't create padding for each element
- Image doesn't have any constrains, meaning if the user added a bigger image that will break your design.